extensions/amp-mustache/0.1/amp-mustache.js

```diff
@@ -33,9 +34,9 @@ const TAG = 'amp-mustache';
  * See {@link https://github.com/janl/mustache.js/}.
  *
  * @private Visible for testing.
- * @extends {BaseTemplate$$module$src$service$template_impl}
+ * @extends {BaseTemplate}
  */
-export class AmpMustache extends AMP.BaseTemplate {
+export class AmpMustache extends BaseTemplate {
```

Comment:
@erwinmombay Is this okay? `AMP.BaseTemplate` is invisible without this change.

Reply:
We cannot do this, it will cause a large increase in file size.

Reply:
This was fixed in https://github.com/ampproject/amphtml/pull/21618/commits/23f71344953a48da94bd9f734a7156daac74589a.